README.md

```diff
@@ -15,25 +15,31 @@ feature:install opennms-cloud-plugin
 ```
 **Configure**
 
+***Import Certificates***
+
+Before mtls can be enabled, we need to import the certificates via OpenNMS Karaf shell:
+```
+opennms-cloud:import-cert /path/to/credentials/credentials.zip
+```
+The zip will be deleted automatically after a successful import.
+
+***Properties***
+
 The default configuration has the following settings:
 ```
 host=localhost
 port=5001
 tokenKey=x-scope-orgid
```

Comment:
`tokenKey` should be `token` by default. Only for testing purposes, it can be ` x-scope-orgid`; for instance, JUnit tests or when accessing the gRPC Gateway directly without an Ingress doing mTLS in the middle.

Reply:
fixed

Reply:
fixed